libc/config/baremetal/arm/entrypoints.txt

```diff
@@ -26,6 +26,9 @@ set(TARGET_LIBC_ENTRYPOINTS
     # errno.h entrypoints
     libc.src.errno.errno
 
+	# search.h entrypoints
+	libc.src.search.lfind
+
```

Comment:
The indentation looks off here. Can you triple check it please?

One thing I highly recommend is modifying your editor to differentiate between tabs and spaces.

Reply:
Resolved by [5946873](https://github.com/llvm/llvm-project/pull/114692/commits/5946873c35a8e8e1f2e23de1e9974d40eb370067)